modules/apps/frontend-js/frontend-js-alert-support-web/npmscripts.config.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+module.exports = {
+	federation: {
+		bridges: [],
+		runLegacyBuild: false,
+	},
+	preset: '@liferay/npm-scripts/src/presets/standard',
```

Comment:
Same comment I did elsewhere about the need of setting the preset... I know you've told me before, but I still can't wrap my head around this 😂 

Maybe we can make this also mergeable if that isn't the case? And if we can't, what's the reason?